Assets/HoloToolkit-Examples/ARCA/Scripts/ColorChanger.cs

```diff
@@ -1,43 +1,65 @@
 ﻿using System;
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.Networking;
 
-public class ColorChanger : NetworkBehaviour
-
+namespace HoloToolkit.ARCapture
 {
-    [SyncVar]
-    Color color;
+    /// <summary>
+    /// Network component that randomly changes the color of its Renderer every given time
+    /// </summary>
+    public class ColorChanger : NetworkBehaviour
```

Comment:
Is this class required for ARCapture or is it more appropriate for it to live in an example/test folder, outside the HoloToolkit?